quarto v1.8.24 (with Quarto Python API)

tylere opened the pull request:
Changes:
- Adds a second output for the Quarto Python API named "python-quarto"
- Adds CI support for Node v20 and v22 (change made by conda-smithy)

<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

I do have some suggestions for making it better though...


For **recipe/meta.yaml**:

 * ℹ️ The recipe is not parsable by parser `conda-souschef (grayskull)`. This parser is not currently used by conda-forge, but may be in the future. We are collecting information to see which recipes are compatible with grayskull.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/17871259459](https://github.com/conda-forge/conda-forge-webservices/actions/runs/17871259459). Examine the logs at this URL for more detail.</sub>

cderv commented:
Hi, 

> Adds a second output for the Quarto Python API named "python-quarto"

Can you explain more what you would like to do ? 

The quarto-cli package (https://pypi.org/project/quarto-cli/) 
```
pip install quarto-cli
```
is a way to install Quarto CLI inside a virtual environment. 

It is not a Python API to Quarto. So as this conda-forge is a way to package quarto CLI for conda, I have a hard time understanding what the installation from pipy is adding to the package

thanks for your clarification.

tylere commented:
> Hi,
> 
> > Adds a second output for the Quarto Python API named "python-quarto"
> 
> Can you explain more what you would like to do ?
> 
> The quarto-cli package (https://pypi.org/project/quarto-cli/)
> 
> ```
> pip install quarto-cli
> ```
> 
> is a way to install Quarto CLI inside a virtual environment.
> 
> It is not a Python API to Quarto. So as this conda-forge is a way to package quarto CLI for conda, I have a hard time understanding what the installation from pipy is adding to the package
> 
> thanks for your clarification.

I am using pixi for package management, and I would like to install the Quarto CLI & Python API. I want to avoid mixing conda and pypi package installs, so I am trying to make a conda-forge package for the Quarto Python API.

The current conda-forge [quarto](https://anaconda.org/conda-forge/quarto) package is not working for me, as it installs only the Quarto CLI, but not the Quarto Python API.

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

mfisher87 commented:
How do you feel about `quarto-python` instead of `python-quarto`? I'd like the packages to be alphabetized together when listing dependencies with e.g. `pixi list`.

tylere commented:
Looks like the remaining build failures need to be resolved manually by the conda-forge folks, by an "admin-requests" trigger.

[Package validation failed for a new output name](https://conda-forge.org/docs/maintainer/infrastructure/#package-validation-failed-for-a-new-output-name)
> New packages that are added to existing feedstocks are not registered automatically in order to prevent typo squatting and other malicious activities. Package outputs are added during feedstock creation. If you move a package from one feedstock to another, **add an output package**, or change the name of a package, **you will need to request that the new package name be added to your feedstock via the [admin-requests repo](https://github.com/conda-forge/admin-requests?tab=readme-ov-file#add-a-package-output-to-a-feedstock)**.

I suspect this request needs to be made by  one of the feedstock owners.

@cderv are you comfortable with extending this feedstock to add a package that installs the Python API (i.e. the changes made by this PR)?

cderv commented:
> are you comfortable with extending this feedstock to add a package that installs the Python API (i.e. the changes made by this PR)?

Honestly, I am a bit lost. What are you calling the `Python API`?  

I must acknowledge I am no conda expert, so adding a package to the feedstock is new to me. And from the [README edit](https://github.com/conda-forge/quarto-feedstock/pull/86/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5), I am not sure to understand as sometimes a `python-quarto` and `quarto` are installed:
https://github.com/tylere/quarto-feedstock/blob/c911f9c3423307bb4f2e5b42b390eb62c94ed3ca/README.md?plain=1#L86

but then some edits replace the listing and search to be only for `python-quarto` 
https://github.com/tylere/quarto-feedstock/blob/c911f9c3423307bb4f2e5b42b390eb62c94ed3ca/README.md?plain=1#L98

https://github.com/tylere/quarto-feedstock/blob/c911f9c3423307bb4f2e5b42b390eb62c94ed3ca/README.md?plain=1#L110-L117

Is `conda search quarto --channel conda-forge` not possible after this change ? 

> The current conda-forge [quarto](https://anaconda.org/conda-forge/quarto) package is not working for me, as it installs only the Quarto CLI, but not the Quarto Python API.

When you say the above, it seems you expect something different than Quarto CLI being install. What is it? What is the Quarto Python API ? 

I would like to understand what is not working for you, and what is missing ? 

Is this just about having the pipy package **quarto** https://pypi.org/project/quarto/ (not [`quarto-cli`](https://pypi.org/project/quarto-cli/)) installed at the same time when installing quarto CLI with this conda forge package? 

I guess the previous script removed in late commit threw me off as they were referring to `quarto-cli`, and this PR is just about having an equivalent in conda-forge for https://pypi.org/project/quarto/, right ?

tylere commented:
> Honestly, I am a bit lost. What are you calling the `Python API`?

Sorry for the ambiguous terminology. I would like to have a conda package that installs the `quarto` Python package defined by: https://github.com/quarto-dev/quarto-python

This should allow me to programmatically render new Quarto documents with a Python script, as described by: 

https://quarto.org/docs/blog/posts/2025-07-24-parameterized-reports-python/index.html#automate-rendering-with-many-parameter-values

> Is this just about having the pipy package quarto https://pypi.org/project/quarto/ (not [quarto-cli](https://pypi.org/project/quarto-cli/)) installed at the same time when installing quarto CLI with this conda forge package?

Yes.

> I guess the previous script removed in late commit threw me off as they were referring to quarto-cli, and this PR is just about having an equivalent in conda-forge for https://pypi.org/project/quarto/, right ?

Yes.

Currently the conda-forge `quarto` package does not install the Python package. This is demonstrated below:

<details>

<summary>Demo of installing the conda-forge quarto package</summary>

```sh
tylere@Tylers-MBP temp % mkdir quarto_conda_test
tylere@Tylers-MBP temp % cd quarto_conda_test 
tylere@Tylers-MBP quarto_conda_test % conda create -n quarto_test      
Retrieving notices: done
Channels:
 - conda-forge
 - defaults
Platform: osx-arm64
Collecting package metadata (repodata.json): done
Solving environment: done


==> WARNING: A newer version of conda exists. <==
    current version: 25.5.1
    latest version: 25.7.0

Please update conda by running

    $ conda update -n base -c conda-forge conda



## Package Plan ##

  environment location: /Users/tylere/.conda/envs/quarto_test



Proceed ([y]/n)? y


Downloading and Extracting Packages:

Preparing transaction: done
Verifying transaction: done
Executing transaction: done
#
# To activate this environment, use
#
#     $ conda activate quarto_test
#
# To deactivate an active environment, use
#
#     $ conda deactivate

tylere@Tylers-MBP quarto_conda_test % conda activate quarto_test
(quarto_test) tylere@Tylers-MBP quarto_conda_test % python
Python 3.12.0 | packaged by conda-forge | (main, Oct  3 2023, 08:36:57) [Clang 15.0.7 ] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import quarto
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'quarto'
>>> exit()
(quarto_test) tylere@Tylers-MBP quarto_conda_test % conda install pip
Channels:
 - conda-forge
 - defaults
Platform: osx-arm64
Collecting package metadata (repodata.json): done
Solving environment: done


==> WARNING: A newer version of conda exists. <==
    current version: 25.5.1
    latest version: 25.7.0

Please update conda by running

    $ conda update -n base -c conda-forge conda



## Package Plan ##

  environment location: /Users/tylere/.conda/envs/quarto_test

  added / updated specs:
    - pip


The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    libexpat-2.7.1             |       hec049ff_0          64 KB  conda-forge
    libmpdec-4.0.0             |       h5505292_0          70 KB  conda-forge
    pip-25.2                   |     pyh145f28c_0         1.1 MB  conda-forge
    python-3.13.7              |h5c937ed_100_cp313        11.4 MB  conda-forge
    python_abi-3.13            |          8_cp313           7 KB  conda-forge
    ------------------------------------------------------------
                                           Total:        12.6 MB

The following NEW packages will be INSTALLED:

  bzip2              conda-forge/osx-arm64::bzip2-1.0.8-hd037594_8 
  ca-certificates    conda-forge/noarch::ca-certificates-2025.8.3-hbd8a1cb_0 
  icu                conda-forge/osx-arm64::icu-75.1-hfee45f7_0 
  libexpat           conda-forge/osx-arm64::libexpat-2.7.1-hec049ff_0 
  libffi             conda-forge/osx-arm64::libffi-3.4.6-h1da3d7d_1 
  liblzma            conda-forge/osx-arm64::liblzma-5.8.1-h39f12f2_2 
  libmpdec           conda-forge/osx-arm64::libmpdec-4.0.0-h5505292_0 
  libsqlite          conda-forge/osx-arm64::libsqlite-3.50.4-h4237e3c_0 
  libzlib            conda-forge/osx-arm64::libzlib-1.3.1-h8359307_2 
  ncurses            conda-forge/osx-arm64::ncurses-6.5-h5e97a16_3 
  openssl            conda-forge/osx-arm64::openssl-3.5.3-h5503f6c_0 
  pip                conda-forge/noarch::pip-25.2-pyh145f28c_0 
  python             conda-forge/osx-arm64::python-3.13.7-h5c937ed_100_cp313 
  python_abi         conda-forge/noarch::python_abi-3.13-8_cp313 
  readline           conda-forge/osx-arm64::readline-8.2-h1d1bf99_2 
  tk                 conda-forge/osx-arm64::tk-8.6.13-h892fb3f_2 
  tzdata             conda-forge/noarch::tzdata-2025b-h78e105d_0 


Proceed ([y]/n)? y


Downloading and Extracting Packages:
                                                                                                    
Preparing transaction: done                                                                         
Verifying transaction: done                                                                         
Executing transaction: done                                                                         
(quarto_test) tylere@Tylers-MBP quarto_conda_test % pip install quarto
Collecting quarto
  Downloading quarto-0.1.0-py3-none-any.whl.metadata (1.4 kB)
Collecting jupyter-core (from quarto)
  Downloading jupyter_core-5.8.1-py3-none-any.whl.metadata (1.6 kB)
Collecting nbformat (from quarto)
  Downloading nbformat-5.10.4-py3-none-any.whl.metadata (3.6 kB)
Collecting nbclient (from quarto)
  Downloading nbclient-0.10.2-py3-none-any.whl.metadata (8.3 kB)
Collecting ipykernel (from quarto)
  Downloading ipykernel-6.30.1-py3-none-any.whl.metadata (6.2 kB)
Collecting pyyaml (from quarto)
  Using cached PyYAML-6.0.2-cp313-cp313-macosx_11_0_arm64.whl.metadata (2.1 kB)
Collecting appnope>=0.1.2 (from ipykernel->quarto)
  Downloading appnope-0.1.4-py2.py3-none-any.whl.metadata (908 bytes)
Collecting comm>=0.1.1 (from ipykernel->quarto)
  Downloading comm-0.2.3-py3-none-any.whl.metadata (3.7 kB)
Collecting debugpy>=1.6.5 (from ipykernel->quarto)
  Downloading debugpy-1.8.17-py2.py3-none-any.whl.metadata (1.5 kB)
Collecting ipython>=7.23.1 (from ipykernel->quarto)
  Downloading ipython-9.5.0-py3-none-any.whl.metadata (4.4 kB)
Collecting jupyter-client>=8.0.0 (from ipykernel->quarto)
  Downloading jupyter_client-8.6.3-py3-none-any.whl.metadata (8.3 kB)
Collecting matplotlib-inline>=0.1 (from ipykernel->quarto)
  Downloading matplotlib_inline-0.1.7-py3-none-any.whl.metadata (3.9 kB)
Collecting nest-asyncio>=1.4 (from ipykernel->quarto)
  Downloading nest_asyncio-1.6.0-py3-none-any.whl.metadata (2.8 kB)
Collecting packaging>=22 (from ipykernel->quarto)
  Downloading packaging-25.0-py3-none-any.whl.metadata (3.3 kB)
Collecting psutil>=5.7 (from ipykernel->quarto)
  Downloading psutil-7.1.0-cp36-abi3-macosx_11_0_arm64.whl.metadata (23 kB)
Collecting pyzmq>=25 (from ipykernel->quarto)
  Downloading pyzmq-27.1.0-cp312-abi3-macosx_10_15_universal2.whl.metadata (6.0 kB)
Collecting tornado>=6.2 (from ipykernel->quarto)
  Downloading tornado-6.5.2-cp39-abi3-macosx_10_9_universal2.whl.metadata (2.8 kB)
Collecting traitlets>=5.4.0 (from ipykernel->quarto)
  Downloading traitlets-5.14.3-py3-none-any.whl.metadata (10 kB)
Collecting decorator (from ipython>=7.23.1->ipykernel->quarto)
  Downloading decorator-5.2.1-py3-none-any.whl.metadata (3.9 kB)
Collecting ipython-pygments-lexers (from ipython>=7.23.1->ipykernel->quarto)
  Downloading ipython_pygments_lexers-1.1.1-py3-none-any.whl.metadata (1.1 kB)
Collecting jedi>=0.16 (from ipython>=7.23.1->ipykernel->quarto)
  Downloading jedi-0.19.2-py2.py3-none-any.whl.metadata (22 kB)
Collecting pexpect>4.3 (from ipython>=7.23.1->ipykernel->quarto)
  Downloading pexpect-4.9.0-py2.py3-none-any.whl.metadata (2.5 kB)
Collecting prompt_toolkit<3.1.0,>=3.0.41 (from ipython>=7.23.1->ipykernel->quarto)
  Downloading prompt_toolkit-3.0.52-py3-none-any.whl.metadata (6.4 kB)
Collecting pygments>=2.4.0 (from ipython>=7.23.1->ipykernel->quarto)
  Downloading pygments-2.19.2-py3-none-any.whl.metadata (2.5 kB)
Collecting stack_data (from ipython>=7.23.1->ipykernel->quarto)
  Using cached stack_data-0.6.3-py3-none-any.whl.metadata (18 kB)
Collecting wcwidth (from prompt_toolkit<3.1.0,>=3.0.41->ipython>=7.23.1->ipykernel->quarto)
  Downloading wcwidth-0.2.14-py2.py3-none-any.whl.metadata (15 kB)
Collecting parso<0.9.0,>=0.8.4 (from jedi>=0.16->ipython>=7.23.1->ipykernel->quarto)
  Downloading parso-0.8.5-py2.py3-none-any.whl.metadata (8.3 kB)
Collecting python-dateutil>=2.8.2 (from jupyter-client>=8.0.0->ipykernel->quarto)
  Downloading python_dateutil-2.9.0.post0-py2.py3-none-any.whl.metadata (8.4 kB)
Collecting platformdirs>=2.5 (from jupyter-core->quarto)
  Downloading platformdirs-4.4.0-py3-none-any.whl.metadata (12 kB)
Collecting ptyprocess>=0.5 (from pexpect>4.3->ipython>=7.23.1->ipykernel->quarto)
  Using cached ptyprocess-0.7.0-py2.py3-none-any.whl.metadata (1.3 kB)
Collecting six>=1.5 (from python-dateutil>=2.8.2->jupyter-client>=8.0.0->ipykernel->quarto)
  Downloading six-1.17.0-py2.py3-none-any.whl.metadata (1.7 kB)
Collecting fastjsonschema>=2.15 (from nbformat->quarto)
  Downloading fastjsonschema-2.21.2-py3-none-any.whl.metadata (2.3 kB)
Collecting jsonschema>=2.6 (from nbformat->quarto)
  Downloading jsonschema-4.25.1-py3-none-any.whl.metadata (7.6 kB)
Collecting attrs>=22.2.0 (from jsonschema>=2.6->nbformat->quarto)
  Downloading attrs-25.3.0-py3-none-any.whl.metadata (10 kB)
Collecting jsonschema-specifications>=2023.03.6 (from jsonschema>=2.6->nbformat->quarto)
  Downloading jsonschema_specifications-2025.9.1-py3-none-any.whl.metadata (2.9 kB)
Collecting referencing>=0.28.4 (from jsonschema>=2.6->nbformat->quarto)
  Using cached referencing-0.36.2-py3-none-any.whl.metadata (2.8 kB)
Collecting rpds-py>=0.7.1 (from jsonschema>=2.6->nbformat->quarto)
  Downloading rpds_py-0.27.1-cp313-cp313-macosx_11_0_arm64.whl.metadata (4.2 kB)
Collecting executing>=1.2.0 (from stack_data->ipython>=7.23.1->ipykernel->quarto)
  Downloading executing-2.2.1-py2.py3-none-any.whl.metadata (8.9 kB)
Collecting asttokens>=2.1.0 (from stack_data->ipython>=7.23.1->ipykernel->quarto)
  Downloading asttokens-3.0.0-py3-none-any.whl.metadata (4.7 kB)
Collecting pure-eval (from stack_data->ipython>=7.23.1->ipykernel->quarto)
  Downloading pure_eval-0.2.3-py3-none-any.whl.metadata (6.3 kB)
Downloading quarto-0.1.0-py3-none-any.whl (10 kB)
Downloading ipykernel-6.30.1-py3-none-any.whl (117 kB)
Downloading appnope-0.1.4-py2.py3-none-any.whl (4.3 kB)
Downloading comm-0.2.3-py3-none-any.whl (7.3 kB)
Downloading debugpy-1.8.17-py2.py3-none-any.whl (5.3 MB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 5.3/5.3 MB 34.0 MB/s  0:00:00
Downloading ipython-9.5.0-py3-none-any.whl (612 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 612.4/612.4 kB 25.4 MB/s  0:00:00
Downloading prompt_toolkit-3.0.52-py3-none-any.whl (391 kB)
Downloading jedi-0.19.2-py2.py3-none-any.whl (1.6 MB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.6/1.6 MB 33.0 MB/s  0:00:00
Downloading parso-0.8.5-py2.py3-none-any.whl (106 kB)
Downloading jupyter_client-8.6.3-py3-none-any.whl (106 kB)
Downloading jupyter_core-5.8.1-py3-none-any.whl (28 kB)
Downloading matplotlib_inline-0.1.7-py3-none-any.whl (9.9 kB)
Downloading nest_asyncio-1.6.0-py3-none-any.whl (5.2 kB)
Downloading packaging-25.0-py3-none-any.whl (66 kB)
Downloading pexpect-4.9.0-py2.py3-none-any.whl (63 kB)
Downloading platformdirs-4.4.0-py3-none-any.whl (18 kB)
Downloading psutil-7.1.0-cp36-abi3-macosx_11_0_arm64.whl (246 kB)
Using cached ptyprocess-0.7.0-py2.py3-none-any.whl (13 kB)
Downloading pygments-2.19.2-py3-none-any.whl (1.2 MB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.2/1.2 MB 41.8 MB/s  0:00:00
Downloading python_dateutil-2.9.0.post0-py2.py3-none-any.whl (229 kB)
Downloading pyzmq-27.1.0-cp312-abi3-macosx_10_15_universal2.whl (1.3 MB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.3/1.3 MB 37.3 MB/s  0:00:00
Downloading six-1.17.0-py2.py3-none-any.whl (11 kB)
Downloading tornado-6.5.2-cp39-abi3-macosx_10_9_universal2.whl (442 kB)
Downloading traitlets-5.14.3-py3-none-any.whl (85 kB)
Downloading decorator-5.2.1-py3-none-any.whl (9.2 kB)
Downloading ipython_pygments_lexers-1.1.1-py3-none-any.whl (8.1 kB)
Downloading nbclient-0.10.2-py3-none-any.whl (25 kB)
Downloading nbformat-5.10.4-py3-none-any.whl (78 kB)
Downloading fastjsonschema-2.21.2-py3-none-any.whl (24 kB)
Downloading jsonschema-4.25.1-py3-none-any.whl (90 kB)
Downloading attrs-25.3.0-py3-none-any.whl (63 kB)
Downloading jsonschema_specifications-2025.9.1-py3-none-any.whl (18 kB)
Using cached referencing-0.36.2-py3-none-any.whl (26 kB)
Downloading rpds_py-0.27.1-cp313-cp313-macosx_11_0_arm64.whl (345 kB)
Using cached PyYAML-6.0.2-cp313-cp313-macosx_11_0_arm64.whl (171 kB)
Using cached stack_data-0.6.3-py3-none-any.whl (24 kB)
Downloading asttokens-3.0.0-py3-none-any.whl (26 kB)
Downloading executing-2.2.1-py2.py3-none-any.whl (28 kB)
Downloading pure_eval-0.2.3-py3-none-any.whl (11 kB)
Downloading wcwidth-0.2.14-py2.py3-none-any.whl (37 kB)
Installing collected packages: pure-eval, ptyprocess, fastjsonschema, wcwidth, traitlets, tornado, six, rpds-py, pyzmq, pyyaml, pygments, psutil, platformdirs, pexpect, parso, packaging, nest-asyncio, executing, decorator, debugpy, comm, attrs, asttokens, appnope, stack_data, referencing, python-dateutil, prompt_toolkit, matplotlib-inline, jupyter-core, jedi, ipython-pygments-lexers, jupyter-client, jsonschema-specifications, ipython, jsonschema, ipykernel, nbformat, nbclient, quarto
Successfully installed appnope-0.1.4 asttokens-3.0.0 attrs-25.3.0 comm-0.2.3 debugpy-1.8.17 decorator-5.2.1 executing-2.2.1 fastjsonschema-2.21.2 ipykernel-6.30.1 ipython-9.5.0 ipython-pygments-lexers-1.1.1 jedi-0.19.2 jsonschema-4.25.1 jsonschema-specifications-2025.9.1 jupyter-client-8.6.3 jupyter-core-5.8.1 matplotlib-inline-0.1.7 nbclient-0.10.2 nbformat-5.10.4 nest-asyncio-1.6.0 packaging-25.0 parso-0.8.5 pexpect-4.9.0 platformdirs-4.4.0 prompt_toolkit-3.0.52 psutil-7.1.0 ptyprocess-0.7.0 pure-eval-0.2.3 pygments-2.19.2 python-dateutil-2.9.0.post0 pyyaml-6.0.2 pyzmq-27.1.0 quarto-0.1.0 referencing-0.36.2 rpds-py-0.27.1 six-1.17.0 stack_data-0.6.3 tornado-6.5.2 traitlets-5.14.3 wcwidth-0.2.14
(quarto_test) tylere@Tylers-MBP quarto_conda_test % python
Python 3.13.7 | packaged by conda-forge | (main, Sep  3 2025, 14:24:46) [Clang 19.1.7 ] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import quarto
>>> exit()
(quarto_test) tylere@Tylers-MBP quarto_conda_test %
```
</details>


> but then some edits replace the listing and search to be only for python-quarto
https://github.com/tylere/quarto-feedstock/blob/c911f9c3423307bb4f2e5b42b390eb62c94ed3ca/README.md?plain=1#L98
>
> https://github.com/tylere/quarto-feedstock/blob/c911f9c3423307bb4f2e5b42b390eb62c94ed3ca/README.md?plain=1#L110-L117
>
> Is conda search quarto --channel conda-forge not possible after this change ?

I did not make all the changes to the README.md file... thanks for catching that. I added a new commit to address this.

> If so, one thing to notice is that we don't release the package version (https://pypi.org/project/quarto/) in same release pace and workflow than the quarto CLI (https://github.com/quarto-dev/quarto-cli).

> I don't know how that worked with this feedstock release process, but possibly quarto-python on conda would not have the same versioning of pypi one. Maybe this is not a concern, but I figured I shared. @mfisher87 you know better than me on this feedstock process, and possible link with quarto release logic

Good observation. This difference in release cadence suggests that there should be two separate conda-forge feedstocks  (quarto-feedstock and quarto-python-feedstock). If that makes sense to everyone, I will abandon this PR and instead start a new feedstock (quarto-python-feedstock).

tylere commented:
I submitted a PR for a new quarto-python-feedstock:
https://github.com/conda-forge/staged-recipes/pull/31079

tylere commented:
Abandoning this PR in favor of [https://github.com/conda-forge/staged-recipes/pull/31079](https://github.com/conda-forge/staged-recipes/pull/31079).
